### PR TITLE
2022q4 corrections and suggestions

### PR DESCRIPTION
**Pull request by grahamperrin**
Most things should be self-explanatory from either the change, or the commit message for the change.

### CODE REVIEW

File: 2022q4/status.adoc
Comment (grahamperrin) [inline 2022q4/status.adoc:49]: @lsalvadore please: was your original intention an indented paragraph, without a bullet point, beneath the bulleted list item?

**Review by AlexanderChernikov — APPROVED** [review]

File: 2022q4/cloud-init.adoc
Comment (pauamma) [inline 2022q4/cloud-init.adoc:6]: OK for removing `link:` between `[]` (which I missed) but only that one.

File: 2022q4/cloud-init.adoc
Comment (pauamma) [inline 2022q4/cloud-init.adoc:4]: That one's OK too.

File: 2022q4/netlink.adoc
Comment (pauamma) [inline 2022q4/netlink.adoc]: That would create a link to something that doesn't actually exist.

**Review by pauamma — CHANGES_REQUESTED** [review]
Overall:
- link: stays except where I OK'd removal
- As I said, I'd like releng approval for that one
- Besides the above, I'm fine with those changes, thanks.

File: 2022q4/cloud-init.adoc
Comment (grahamperrin) [reply 2022q4/cloud-init.adoc:4]: Do you mean, OK _without_ the `link:` prefix in nine places (in the _Links:_ section)?

File: 2022q4/releng.adoc
Comment (grahamperrin) [reply 2022q4/releng.adoc:24]: > I'd like releng approval for this one, …

IMHO there's _obviously_ a markup error in the original: 

image

– written as two separate lines, but rendered without the separation:

>> Sponsor: Rubicon Communications, LLC ("Netgate") Sponsor: The FreeBSD Foundation

**Comment by grahamperrin** [comment]
NB from the intro: 

> Lorenzo Salvadore, on behalf of the status report team.

(Not the _status team_.)

File: 2022q4/freebsd-foundation.adoc
Comment (grahamperrin) [inline 2022q4/freebsd-foundation.adoc:25]: @Jehops unless I'm missing something, there's no Arm64 report for 2022q4: <https://github.com/freebsd/freebsd-quarterly/tree/main/2022q4>. 

Amend the sentence here, or leave as is?

File: 2022q4/freebsd-foundation.adoc
Comment (lsalvadore) [inline 2022q4/freebsd-foundation.adoc:9]: Why are you changing the capitalization for line 7 only?

@Jehops: Do you approve the capitalization change? If not, please note that the actual use of capital letters is inconsistent: there are occurences of "FreeBSDfoundation", "FreeBSDFoundation" and of "freebsdfoundation"

The issue regards the rest of the report as well. And it is also present in precedent reports, but now it is better to focus on this one.

File: 2022q4/papers.adoc
Comment (lsalvadore) [inline 2022q4/papers.adoc:3]: As explained in another comment, I would prefer to keep the plural form here.

File: 2022q4/pot.adoc
Comment (lsalvadore) [inline 2022q4/pot.adoc:3]: As explained in another comment, I would prefer to keep the plural form here.

File: 2022q4/riscv.adoc
Comment (lsalvadore) [inline 2022q4/riscv.adoc:15]: @mhorne: I know nothing about riscv and what has been done with it, could you please confirm if changing riscv with riscv64 is a corret and useful in this case?

File: 2022q4/ufs_snapshots.adoc
Comment (lsalvadore) [inline 2022q4/ufs_snapshots.adoc:3]: As explained in another comment, I would prefer to keep the plural form here.

File: 2022q4/vessel-status-report.adoc
Comment (lsalvadore) [inline 2022q4/vessel-status-report.adoc:3]: As explained in another comment, I would prefer to keep the plural form here.

File: 2022q4/xfce.adoc
Comment (lsalvadore) [inline 2022q4/xfce.adoc:9]: Please revert this change. As you can see in other reports, we usually write contacts in the form that was originally proposed.

It might be a good idea to improve how we manage links and contacts and we will probably do it at some point, but as you know we are already busy improving other aspects of our workflow and I do not think we have the time to deal with this too for this quarter.

**Review by lsalvadore — CHANGES_REQUESTED** [review]
Thanks @grahamperrin, for your review. However, please avoid creating huge pull requests that change a lot of files at once, unless they are related changes: it makes reviewing harder. It is easier to have one pull request per report when possible.

In particular, having one pull request per report will be easier for report submitters that we want to contact again so that they do not need to dive into the whole review to find the spot where their attention is requested.

@hselasky, @Jehops, @cperciva, @mhorne: I have written a few comments about the reports you have submitted. If you can, please take a look at them. Thanks.

File: 2022q4/riscv.adoc
Comment (mhorne) [reply 2022q4/riscv.adoc:15]: Confirmed. Either form is acceptable when referencing the FreeBSD architecture support, but riscv64 is slightly more specific so I suggested this one to @grahamperrin.

File: 2022q4/freebsd-foundation.adoc
Comment (Jehops) [reply 2022q4/freebsd-foundation.adoc:9]: Looks good.

Comment (lsalvadore) [reply 2022q4/freebsd-foundation.adoc:9]: Thanks @Jehops.

@grahamperrin: Could you please proceed to change all links in this report to remove the capitalization? I think you can leave the capitalization for the email addresses, just as we usually do for FreeBSD.org addresses.

File: 2022q4/riscv.adoc
Comment (lsalvadore) [reply 2022q4/riscv.adoc:15]: Thanks for your confirmation @mhorne.

**Comment by hselasky** [comment]
On 1/20/23 16:46, Lorenzo Salvadore wrote:
> -Sponsor: NVIDIA Networking (for the kernel development part)
> +Sponsor: NVIDIA(R) Networking (for the kernel development part)
> 
> I have looked for similar lines in commits messages (more precisely, I have run `git log --grep=NVIDIA | grep "NVIDIA"` in the FreeBSD src repository) and I have only found "NVIDIA Networking", without any "(R)". So I would revert this change, or at least I would get explicit approval from the report author ***@***.***)  who is probably the person who knows better how the sponsorization needs to be reported.

I've just been told to use "Sponsored by: NVIDIA Networking", when I 
submit stuff like this. Technically I don't know about the (R). NVIDIA 
Networking, is an own group inside NVIDIA. Probably you should write it 
without (R), or if it is a problem, use hselasky@ / NVIDIA Networking .


Regarding the other changes, they look OK.

--HPS

File: 2022q4/accessibility.adoc
Comment (lsalvadore) [reply 2022q4/accessibility.adoc:22]: Thanks @hselasky. I think then it is better to remove the (R) and come back to the original form that is being used consistently in commit messages.

File: 2022q4/releng.adoc
Comment (cperciva) [reply 2022q4/releng.adoc:24]: Sorry, I didn't notice this last week.  I'm fine with either fix.

Comment (lsalvadore) [reply 2022q4/releng.adoc:24]: Ok, thanks. Then we keep @grahamperrin's change as it is.

**Comment by grahamperrin** [comment]
Thanks, everyone. I'll get to all of the above later today. 

> … easier to have one pull request per report when possible. …

Thanks @lsalvadore, I will do in future. (I assumed, wrongly, that pros of a cohesive PR would outweigh the cons.)

File: 2022q4/status.adoc
Comment (grahamperrin) [reply 2022q4/status.adoc:26]: Thanks. Resolved with [`a8f29a1` (#575)](https://github.com/freebsd/freebsd-quarterly/pull/575/commits/a8f29a10b5a1fd62630e15a1cbfd9c2a1d96a91b)

File: 2022q4/intro.adoc
Comment (grahamperrin) [reply 2022q4/intro.adoc]: Thanks. Resolved with [`a8f29a1` (#575)](https://github.com/freebsd/freebsd-quarterly/pull/575/commits/a8f29a10b5a1fd62630e15a1cbfd9c2a1d96a91b)

File: 2022q4/accessibility.adoc
Comment (grahamperrin) [reply 2022q4/accessibility.adoc:6]: Consistency, including: 

* no uppercase for _console_ or _kernel_ in the preceding lines
* _base system_ without uppercase in the linked wiki page.

File: 2022q4/status.adoc
Comment (grahamperrin) [reply 2022q4/status.adoc]: Thanks. Resolved with [`30780e3` (#575)](https://github.com/freebsd/freebsd-quarterly/pull/575/commits/30780e343c921e1f31c15b5c4bc5c9960a1bf1dd)

File: 2022q4/accessibility.adoc
Comment (grahamperrin) [reply 2022q4/accessibility.adoc]: Thanks. Resolved with [`91ea422` (#575)](https://github.com/freebsd/freebsd-quarterly/pull/575/commits/91ea4225f7ab0bd730c314e265f36ad03a393b07)

File: 2022q4/xfce.adoc
Comment (grahamperrin) [reply 2022q4/xfce.adoc]: Reverted with [`44d75b7` (#575)](https://github.com/freebsd/freebsd-quarterly/pull/575/commits/44d75b79a3374d728a2120999adb79afdb5f2319). The result, previewed: 

> Contact: Xfce team <[xfce@FreeBSD.org](mailto:xfce@FreeBSD.org)> Contact: Guido Falsi <[madpilot@FreeBSD.org](mailto:madpilot@FreeBSD.org)>

– multiple contacts on a single line. Please, was this the intention?

File: 2022q4/vessel-status-report.adoc
Comment (grahamperrin) [reply 2022q4/vessel-status-report.adoc]: Hopefully resolved with [`3b89ec6` (#575)](https://github.com/freebsd/freebsd-quarterly/pull/575/commits/3b89ec6054deadf4c750d73183bfe72d7bf34db2).

Note that I did not reintroduce the word "for".

File: 2022q4/budgie.adoc
Comment (grahamperrin) [reply 2022q4/budgie.adoc]: Thanks. 

[`f2a0e8a` (#575)](https://github.com/freebsd/freebsd-quarterly/pull/575/commits/f2a0e8a37dd5e20bb90581a2852a52fe182ac86a) also reverted _two_ to _2_. OK?

File: 2022q4/status.adoc
Comment (grahamperrin) [reply 2022q4/status.adoc:53]: Can you clarify what's required here? Thanks.

**Comment by grahamperrin** [comment]
> … NVIDIA Networking, … Probably you should write it without (R), …

Thanks @hselasky, resolved with [`91ea422` (#575)](https://github.com/freebsd/freebsd-quarterly/pull/575/commits/91ea4225f7ab0bd730c314e265f36ad03a393b07)

**Comment by grahamperrin** [comment]
> Overall:
> 
>     * link: stays except where I OK'd removal
> 
>     * As I said, I'd like releng approval for that one
> 
>     * Besides the above, I'm fine with those changes, thanks.

Thanks @pauamma, I think that your request for changes is fulfilled.

File: 2022q4/vessel-status-report.adoc
Comment (grahamperrin) [reply 2022q4/vessel-status-report.adoc]: For the future: phrases such as _thin jail_, _thick jail_ and _fat jail_ – without quotation marks – are seen often enough for me to believe that they're OK without. 

Examples: 

* [Why I prefer thick jails over thin jails :: FreeBSD Presentations and Papers](https://papers.freebsd.org/2019/eurobsdcon/dvl-why_i_prefer_thick_jails_over_thin_jails/) @dlangille 
* <https://issue.freebsdfoundation.org/publication/?m=33057&i=616343&p=10&ver=html5> (Michael W. Lucas), foot of the page
* <https://www.startpage.com/do/dsearch?query=%22thin+jail%22+%2B%22FreeBSD%22&cat=web&language=english> various search results.

File: 2022q4/intro.adoc
Comment (grahamperrin) [reply 2022q4/intro.adoc]: NB: 

* _quarterly status reports_ under <https://github.com/grahamperrin/freebsd-quarterly/blob/42e7164c3c324266fc55efe8795875272f33824c/2022q4/status.adoc#team-naming> – maybe change to _status reports_
* _Status reports GitHub repository_ a.k.a. _status repository_ – should the latter change to _status reports repository_?

**Comment by pauamma** [comment]
I looked at the stylistic and markup changes again and nothing jumps out to me, so if/once everyone else mentioned agrees, this PR's fine with me. Noting that the single nit "Python man:atf[7]support" (missing space after ]) this would introduce in 2022q4/pytest-for-atf.adoc can be fixed while/after merging.

**Review by igalic — APPROVED** [review]
okay from me / cloud-init

File: 2022q4/status.adoc
Comment (lsalvadore) [reply 2022q4/status.adoc:53]: I am asking to change `<status-submissions@FreeBSD.org>` into `` `status-submissions@FreeBSD.org` ``, as you did for other email addresses, for example at line 39.

File: 2022q4/accessibility.adoc
Comment (lsalvadore) [reply 2022q4/accessibility.adoc:6]: It makes sense, thanks.

**Comment by lsalvadore** [comment]
I approve as well once the last issue about the email address in status.adoc is fixed.

File: 2022q4/status.adoc
Comment (grahamperrin) [reply 2022q4/status.adoc]: Resolved with [`bbe5153` (#575)](https://github.com/freebsd/freebsd-quarterly/pull/575/commits/bbe5153dc59a8ea346d1c944a7875b7f5753a993).

File: 2022q4/intro.adoc
Comment (lsalvadore) [reply 2022q4/intro.adoc]: I think we can leave those two things as they are:

- status.adoc is explaining a name transition and it is natural to reference the old thing with its old name, and the old name often contained the reference to frequency;
- I do not see any reason for removing the fact that we are talking about the repository hosted on GitHub. On the contrary, the specification should help making things clearer.

**Review by lsalvadore — APPROVED** [review]

File: 2022q4/xfce.adoc
Comment (grahamperrin) [reply 2022q4/xfce.adoc]: > … multiple contacts on a single line. Please, was this the intention?

Resolved with https://github.com/freebsd/freebsd-quarterly/commit/94489ef7562bd54ef71f098067be3d83b82b46fa?diff=unified#diff-8bd6d75de486569e8357dec0b6ad372cc892501eba032ca9c485a8cb276b3421

**Pull request merged**
